dev/packages/alpha/cisco/0.0.3/dataset/asa/elasticsearch/ingest-pipeline/default.yml

```diff
@@ -0,0 +1,1296 @@
+---
+description: "Pipeline for Cisco ASA logs"
```

Comment:
@cjcenizal If you want to see a really complex ingest pipeline :-)

Reply:
Thanks @ruflin! This is very helpful. Who authored this pipeline? I'd like to learn about their developer experience to better inform our decisions building the ingest node pipelines app.

Reply:
I think @alakahakai is the person to talk to.